### PR TITLE
CVE-2019-3865 update

### PR DESCRIPTION
**Pull request by darunesh-RH**

### CODE REVIEW

File: 2019/3xxx/CVE-2019-3865.json [hunk 1]
```diff
@@ -1,18 +1,71 @@
...
+                {
+                    "vendor_name": "[UNKNOWN]",
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "quay",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_value": "n/a"
```
Comment (cve-team) [inline 2019/3xxx/CVE-2019-3865.json]: Missing version information

Comment (cristina479) [reply 2019/3xxx/CVE-2019-3865.json]: Include the affected version in the version_data field.

**Comment by cristina479** [comment]
Dear darunesh-RH,

Can you include the requested changes to the `version_data` field?

File: 2019/3xxx/CVE-2019-3865.json (hunk 1 above)
Comment (darunesh-RH) [reply 2019/3xxx/CVE-2019-3865.json]: Fixed version value, affected version is quay 2.

**Pull request merged**